<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MqttsnClientFilterConfigWidget</class>
 <widget class="QWidget" name="MqttsnClientFilterConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="m_retryPeriodLabel">
       <property name="text">
        <string>Retry Period (ms):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_retryPeriodSpinBox">
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="m_retryPeriodLabel_2">
       <property name="text">
        <string>Retry Count:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_retryCountSpinBox">
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_clientIdLabel">
       <property name="toolTip">
        <string>Updated by &quot;mqttsn.client&quot; or &quot;mqtt.client&quot; inter-plugin configuration.</string>
       </property>
       <property name="text">
        <string>Client ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_clientIdLineEdit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="m_keepAliveLabel">
       <property name="text">
        <string>Keep Alive:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_keepAliveSpinBox">
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="m_cleanSessionLabel">
       <property name="toolTip">
        <string>Force clean start on every (re) connection</string>
       </property>
       <property name="text">
        <string>Force Clean Session:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_cleanSessionComboBox">
       <item>
        <property name="text">
         <string>No</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yes</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="m_pubTopicLabel">
       <property name="toolTip">
        <string>Updated by &quot;mqttsn.pub_topic&quot; or &quot;mqtt.pub_topic&quot; inter-plugin configuration</string>
       </property>
       <property name="text">
        <string>Default Publish Topic:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_pubTopicLineEdit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="m_pubTopicIdLabel">
       <property name="toolTip">
        <string>Updated by &quot;mqttsn.pub_qos&quot; or &quot;mqtt.pub_qos&quot; inter-plugin configuration</string>
       </property>
       <property name="text">
        <string>Default Publish Topic ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_pubTopicIdSpinBox">
       <property name="maximum">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="m_pubQosLabel">
       <property name="toolTip">
        <string>Updated by &quot;mqttsn.pub_qos&quot; or &quot;mqtt.pub_qos&quot; inter-plugin configuration</string>
       </property>
       <property name="text">
        <string>Default Publish QoS:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_pubQosSpinBox">
       <property name="maximum">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="m_subsWidget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QPushButton" name="m_addSubPushButton">
       <property name="text">
        <string>Add Subscribe</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
